molo/core/tests/wagtail_hooks/test_delete_page_translations.py

```diff
@@ -57,14 +52,13 @@ def test_deletes_all_on_post_request(self):
         self.assertEqual(ArticlePage.objects.filter(title='En').count(), 0)
         self.assertEqual(ArticlePage.objects.filter(title='Xh').count(), 0)
         self.assertEqual(ArticlePage.objects.filter(title='Zu').count(), 0)
-        self.assertEqual(PageTranslation.objects.all().count(), 0)
```

Comment:
shouldn't this test that there aren't any translated pages now?

Reply:
@KaitCrawford it can't since all the pages are deleted. There is no object left to test against